.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: go
+
+go:
+  - 1.6.x
+  - 1.7.x
+  - 1.8.x
+  - 1.9.x
+  - tip
+
+script:
```

Comment:
Can you add `go_import_path: github.com/mitchellh/gox` so it will also work with PR's from community repos? https://docs.travis-ci.com/user/languages/go/#Go-Import-Path

Reply:
@ruflin Why would PRs from community repos fail without it?

Reply:
@mattfarina Sorry, I think community PR's are fine. But if you work on a fork and also have travis running on it for yourself, this addition is needed.

Reply:
@ruflin can you make a PR for the change?

Reply:
Sure: https://github.com/mitchellh/gox/pull/97